ertviz/models/ensemble_model.py

```diff
@@ -39,18 +39,19 @@ def get_parameter_models(parameters_schema):
 
 class EnsembleModel:
     def __init__(self, ref_url):
-        schema = get_schema(api_url=ref_url)
-        self._name = schema["name"]
+        self._schema = get_schema(api_url=ref_url)
+        self._name = self._schema["name"]
         self._id = ref_url  # ref_url
```

Comment:
seems like an unnecessary comment?

Reply:
It was meant to be a bit more descriptive. For now the unique identifier for things are the address, for it to be fetched from the api. It might be that at one point we are using a more structured http api and then maybe it makes more sense to pass in the id in the schema, and construct http calls based on those ids. Removing the comment for now.